config/main.py

```diff
@@ -3077,5 +3166,355 @@ def autorestart(container_name, autorestart_status):
 
     config_db.mod_entry('CONTAINER_FEATURE', container_name, {'auto_restart': autorestart_status})
 
+#
+# 'vxlan' group ('config vxlan ...')
+#
+@config.group()
+@click.pass_context
+def vxlan(ctx):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {'db': config_db}
+
+@vxlan.command('add')
+@click.argument('vxlan_name', metavar='<vxlan_name>', required=True)
+@click.argument('src_ip', metavar='<src_ip>', required=True)
+@click.pass_context
+def add_vxlan(ctx, vxlan_name, src_ip):
+    """Add VXLAN"""
+    if not is_ip_addr_valid(src_ip, True):
+        ctx.fail("{} invalid src ip address".format(src_ip))  
+    db = ctx.obj['db']
+
+    vxlan_keys = db.keys('CONFIG_DB', "VXLAN_TUNNEL|*")
+    if not vxlan_keys:
+      vxlan_count = 0
+    else:
+      vxlan_count = len(vxlan_keys)
+
+    if(vxlan_count > 0):
+        ctx.fail("VTEP already configured.")  
+
+    fvs = {'src_ip': src_ip}
+    db.set_entry('VXLAN_TUNNEL', vxlan_name, fvs)
+
+@vxlan.command('del')
+@click.argument('vxlan_name', metavar='<vxlan_name>', required=True)
+@click.pass_context
+def del_vxlan(ctx, vxlan_name):
+    """Del VXLAN"""
+    db = ctx.obj['db']
+
+    vxlan_keys = db.keys('CONFIG_DB', "EVPN_NVO|*")
+    if not vxlan_keys:
+      vxlan_count = 0
+    else:
+      vxlan_count = len(vxlan_keys)
+
+    if(vxlan_count > 0):
+        ctx.fail("Please delete the EVPN NVO configuration.")  
+
+    vxlan_keys = db.keys('CONFIG_DB', "VXLAN_TUNNEL_MAP|*")
+    if not vxlan_keys:
+      vxlan_count = 0
+    else:
+      vxlan_count = len(vxlan_keys)
+
+    if(vxlan_count > 0):
+        ctx.fail("Please delete all VLAN VNI mappings.")  
+
+    db.set_entry('VXLAN_TUNNEL', vxlan_name, None)
+
+@vxlan.group('evpn_nvo')
+@click.pass_context
+def vxlan_evpn_nvo(ctx):
+    pass
+
+@vxlan_evpn_nvo.command('add')
+@click.argument('nvo_name', metavar='<nvo_name>', required=True)
+@click.argument('vxlan_name', metavar='<vxlan_name>', required=True)
+@click.pass_context
+def add_vxlan_evpn_nvo(ctx, nvo_name, vxlan_name):
+    """Add NVO"""
+    db = ctx.obj['db']
+    vxlan_keys = db.keys('CONFIG_DB', "EVPN_NVO|*")
```

Comment:
In sonic-swss-commin [PR339](https://github.com/Azure/sonic-swss-common/pull/339/), this table is named as VXLAN_EVPN_NVO

Reply:
Changed to be in sync with PR 339